e2e/testcases/cli_test.go

```diff
@@ -1068,3 +1068,46 @@ func TestNomosVersion(t *testing.T) {
 		nt.T.Fatalf("Expected to find config-sync component in output:\n%s\n", string(out))
 	}
 }
+
+func TestNomosStatusNameFilter(t *testing.T) {
+	nt := nomostest.New(t, nomostesting.NomosCLI)
+
+	// get status
+	cmd := nt.Command("nomos", "status", "--name", "root")
```

Comment:
The RootSync name is `root-sync`, not `root`.

Reply:
this was written with Strings.contains in mind, fixed now